backend/core/src/application/application.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { ClassSerializerInterceptor, Controller, Get, Query, UseInterceptors } from "@nestjs/common"
+import { ApplicationListQueryParams } from "./application.dto"
+import { ApplicationService } from "./application.service"
+import { plainToClass } from "class-transformer"
+import { Application } from "../entity/application.entity"
+
+@Controller("application")
+export class ApplicationController {
```

Comment:
For consistency (and also because Application can have many meanings), can we rename this to `ApplicationsController` (plural) like we have `ListingsController` (and same for the service)?

Reply:
Done